pkg/clients/integration/integration_test_scenarios.go

```diff
@@ -63,10 +63,15 @@ func (i *IntegrationController) CreateIntegrationTestScenarioWithEnvironment(app
 }
 
 // CreateIntegrationTestScenario creates beta1 version integrationTestScenario.
-func (i *IntegrationController) CreateIntegrationTestScenario(applicationName, namespace, gitURL, revision, pathInRepo string) (*integrationv1beta1.IntegrationTestScenario, error) {
+func (i *IntegrationController) CreateIntegrationTestScenario(itsName, applicationName, namespace, gitURL, revision, pathInRepo string) (*integrationv1beta1.IntegrationTestScenario, error) {
+//func (i *IntegrationController) CreateIntegrationTestScenario(itsName, applicationName, additionalParams ...string) (*integrationv1beta1.IntegrationTestScenario, error) {
```

Comment:
Can we remove this comment?

Reply:
Oh, forgot about that. Fixed now, thank you.